README.md

```diff
@@ -37,9 +38,10 @@ The first time you run the utility, you will be prompted to allow the terminal e
 
 ![Terminal wants access to control](screenshots/allow-control-1.png)
 
-This is because of my implementation approach of cycling through the desktops by sending Ctrl+Arrow keys. Click Ok then add your Terminal or iTerm to the list of allowed applications:
+This is because of my implementation approach of cycling through the desktops by sending Ctrl+Arrow keys. Click Ok then add your Terminal or iTerm as well as adding Apple's Script Editor to the list of allowed applications:
```

Comment:
Edit sentence for clarity. Should say something like "Click Ok then add your Terminal emulator and Apple's Script Editor to the list of allowed applications"